packages/tree-view/src/__stories__/stories.js

```diff
@@ -21,6 +23,30 @@ import {
 import { TreeItem } from "../index";
 import sampleTreeNodeObject from "../__fixtures__/sampleTreeNodeObject";
 
+// test that the TreeView can render a high-order TreeItem component or a function TreeItem component
+export class HocTreeItem extends React.Component {
```

Comment:
Hello! Thank you for your submission. Is there something else we can call this that's not an acronym? Readability for open source libraries is really important and I'm not even really sure what HOC means (except after reading the code).